consensus/propagation/catchup_test.go

```diff
@@ -66,7 +66,7 @@ func TestGapCatchup(t *testing.T) {
 
 	// this call simulates getting a commitment for a proposal of a higher
 	// height
-	n2.retryWants(2)
+	n2.retryWants(prop.Height, prop.Round)
```

Comment:
this should be changed to better tests where we try:

- catchup multiple heights at the same time
- retry the same height